x-pack/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/calendars/description.tsx

```diff
@@ -14,8 +14,7 @@ export const Description: FC = memo(({ children }) => {
   const {
     services: { docLinks },
   } = useMlKibana();
-  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
-  const docsUrl = `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-calendars.html`;
+  const docsUrl = docLinks.links.ml.customUrls;
```

Comment:
Copy and paste error? This should be `docLinks.links.ml.calendars` I think.

Reply:
oops, yup, I'll fix that.